src/index.ts

```diff
@@ -1,4 +1,15 @@
-export * from './domain-functions.ts'
+export {
```

Comment:
If we drop the `oldMerge` export this change would become optional. Having said that, having an explicit list of exports does not bother me one bit.

Reply:
I thought the same. I think we can reduce the diff by not exporting the old merge.